README.md

```diff
@@ -21,12 +21,34 @@
 ## What does not work
 - Master creation (Not planned at the moment)
 
-# Running
+# Quickstart
 
-## Requirements
-The controller expects a cluster-info configmap to exist within the kube-public namespace.
-This configmap needs to contain a kubeconfig without credentials. It is already present in all kubeadm-generated
-clusters.
+## Deploy the machine-controller
+
+`kubectl apply -f examples/machine-controller.yaml`
+
+## Creating a machine
+```bash
+# edit examples/machine.yaml & create the machine
+kubectl create -f examples/machine.yaml
+```
+
+## Advanced usage
+
+### Specifying the apiserver endpoint
+By default the controller looks for a cluster-info ConfigMap within the kube-public Namespace.
+If one is found which contains a minimal kubeconfig (kubeadm cluster have them by default), this kubeconfig will be used for the node bootstrapping.
+The kubeconfig only needs to container only two things:
+- CA-Data
+- The public endpoint for the Apiserver 
+
+If no ConfigMap can be found:
```

Comment:
You need to insert newlines before and after `**CA-data**`/`**Apiserver endpoint**`, otherwise the markdown rendering puts "If no ConfigMap can be found: **CA-data** " into the same line. (I usually just go to the repo and select the branch to see the rendered markdown and verify it looks as intended).